front-end-side/src/Pages/Friends.css

```diff
@@ -0,0 +1,6 @@
+body {
+}
```

Comment:
Why you add body tag if it's empty?

Reply:
It was automatically created, I will delete it, sorry.